public/js/app/components-shared/Alert.jsx

```diff
@@ -16,8 +16,4 @@ Alert.propTypes = {
   type: PropTypes.oneOf(['info', 'warning', 'success']).isRequired
```

Comment:
```suggestion
  type: PropTypes.oneOf(['info', 'warning', 'success'])
```

`.isReqired` can be removed since the default value of the incoming param is already set to a value (which marks that it's required).